### DISCUSSION TITLE
Issue in "Search by Album"

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#303] [Fixed Issues]
**rAudio 1 • 20210414**

In the environment of classical music it is not uncommon that you have different recordings of the same work. The album title of both recordings is the same, but the other tags differ e.g. the artist.
Example:
- Album Title: "Ludwig van Beethoven: Sinfonie No 1 C-Dur, Op. 21"
- Artist 1: "Jan Willem de Vriend + The Netherlands Symphony Orchestra (Orkest van het Oosten)"
- Artist 2: "Paavo Järvi + Deutsche Kammerphilharmonie Bremen"

If I browse through the album coverarts choosing e.g. the recording of Artist 1 than both recordings are contained in the playlist not as expected the recording of Artist 1 only. This issue is restricted to the album search: If I select the recording of Artist 1 using tree traversal you experience the expected behaviour.

**Comment by klassikmann** [comment]
**rAudio 1 • 20210416**

fyi: Issue still present.

**Reply by rern** [reply]
Album mode gets a track list by directory.
What are their lines in `/srv/http/data/mpd/album`?

**Reply by klassikmann** [reply]
- Artist 1: "Jan Willem de Vriend + The Netherlands Symphony Orchestra (Orkest van het Oosten)"
`L^^Ludwig van Beethoven: Sinfonie No 1 C-Dur, Op. 21^^Jan Willem de Vriend + The Netherlands Symphony Orchestra (Orkest van het Oosten)^^NAS/Musik/Klassik+Romantik/Beethoven, Ludwig van/Sinfonien/Sinfonie No 1 C-Dur, Op. 21/Jan Willem de Vriend + The Netherlands Symphony Orchestra`
- Artist 2: "Paavo Järvi + Deutsche Kammerphilharmonie Bremen"
`L^^Ludwig van Beethoven: Sinfonie No 1 C-Dur, Op. 21^^Paavo Järvi + Deutsche Kammerphilharmonie Bremen^^NAS/Musik/Klassik+Romantik/Beethoven, Ludwig van/Sinfonien/Sinfonie No 1 C-Dur, Op. 21/Paavo Järvi + Deutsche Kammerphilharmonie Bremen`

**Reply by rern** [reply]
Try the UPDATE branch.

**Reply by klassikmann** [reply]
> Try the UPDATE branch.

Sorry, not resolved. Same effect.

**Reply by rern** [reply]
Try again. It should be OK now.

**Reply by klassikmann** [reply]
> Try again. It should be OK now.

Indeed. Thank you.
